added CSI-4446 as new known issue (#548)
IBM/ibm-block-csi-driver: docs/content/release_notes/known_issues.md

|Ticket ID|Severity|Description|
|---------|--------|-----------|

> |**CSI-4446**|Service|In rare cases, the HostDefiner labels are not deleted during csinode deletion from the nodes. This occurs even when the `allowDelete` parameter is set to `true`.<br>**Workaround:** Manually delete the hosts from the storage system.

Reviewer: ```suggestion
|**CSI-4446**|Service|In rare cases, the `hostdefiner.block.csi.ibm.com/manage-node=true` labels are not deleted during csinode deletion from the nodes. This occurs even when the `allowDelete` 
 and `dynamicNodeLabeling` parameters are set to `true`.<br>**Workaround:** Manually delete the hosts from the storage system.
```

Author: updated

> |**CSI-3382**|Service|After CSI Topology label deletion, volume provisioning does not work, even when not using any topology-aware YAML files.<br>**Workaround:** To allow volume provisioning through the CSI driver, delete the operator pod. <br>After the deletion, a new operator pod is created and the controller pod is automatically restarted, allowing for volume provisioning.|

Reviewer: could you align the spaces between the issues?
